### PR TITLE
Remove kernel coverage checks in FV, add some newsletters, some misc  doco fixes

### PR DESCRIPTION
**Pull request by GiudGiud**

### CODE REVIEW

File: modules/doc/content/newsletter/2021_11.md [hunk 1]
```diff
@@ -67,3 +90,8 @@ If any issues with the new packages are experienced, please open a new
   version variable.
 
 ## Bug Fixes and Minor Enhancements
+
+- Missing user documentation was added for initial conditions, time steppers, kernels, mesh generators,
+  postprocessors, vector postprocessors and nodal kernels.
+- Function-based initial conditions were added for the conserved variable formulation of
+  the fully compressible Navier Stokes equations
```
Comment (lindsayad) [inline modules/doc/content/newsletter/2021_11.md:97]: does this apply to both finite volume and finite element?

File: modules/doc/content/newsletter/2021_11.md [hunk 2]
```diff
@@ -41,6 +41,29 @@ If any issues with the new packages are experienced, please open a new
...
+
+- time derivative of density for the mass equation
+- additional time derivative of density, multiplied by velocity, in the momentum equations
+- additional time derivative of density, multiplied by `cp T`, in the energy equation
+
+An additional kernel was added to handle the mixing length turbulence model without
+having to normalize the energy equation by density, as this was no longer simple with
+a time and space varying fluid density.
+
+All over kernels are shared between the incompressible and weakly compressible formulations.
```
Comment (lindsayad) [inline modules/doc/content/newsletter/2021_11.md:58]:
```suggestion
All other kernels are shared between the incompressible and weakly compressible formulations.
```

**Review by lindsayad — CHANGES_REQUESTED** [review]

File: modules/doc/content/newsletter/2021_11.md (hunk 1 above)
Comment (GiudGiud) [reply modules/doc/content/newsletter/2021_11.md:97]: I think so. They both use material properties, and they both can use initialization of conserved through primitive values.
I think NSIC existed before I added to it too

**Review by lindsayad — DISMISSED** [review]

**Comment by GiudGiud** [comment]
@lindsayad there was a merge conflict on newsletter, need new approval

**Review by lindsayad — APPROVED** [review]

**Pull request merged**